commit 82845c75f87cde61899d7526551934b3c3d634ff (refs/changes/73/3673/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-03-17T00:36:47+00:00 (1 year, 7 months ago)
    
    TRITON-75 Allow firewall rules to be applied in order of priority
    TRITON-77 Add support for ESP and AH protocols in firewall rules

diff --git a/config/sapi/services/fwapi/service.json b/config/sapi/services/fwapi/service.json
index cf220ff1..0fcddcdf 100644
--- a/config/sapi/services/fwapi/service.json
+++ b/config/sapi/services/fwapi/service.json
@@ -17,9 +17,9 @@
         ]
     },
     "metadata": {
-        "FWRULE_VERSION": 3,
+        "FWRULE_VERSION": 4,
         "SERVICE_NAME": "fwapi",
         "FWAPI_LOG_LEVEL": "info"
     },
     "manifests": {}
-}
\ No newline at end of file
+}
